### DISCUSSION TITLE
OPCUA server method implementation example

### DISCUSSION THREAD

**Discussion by dekamerone** [BiancoRoyal/node-red-contrib-iiot-opcua#274] [Q&A]
Hello together,

just a question about OPC-UA server method implementation:
Is it possible to implement OPC-UA server methods in Node-Red with the node-red-contrib-iiot-opcua@4.1.2 module?

Just like the Bark method example ... isn't it?
And if so, how to get started with own implementations?

Many thanks for all help: Michael

**Comment by biancode** [comment]
Hello Micheal, the Methods are not really available in the package and it will be deprecated soon. See our very new P4NR packages to get a massive boost for OPC UA Server development https://plus4nodered.com/ - It includes custom methods and custom types in parameters and direct working with our much easier Web-App framework for node-red and the new LeanPub books will help you to learn how.
Request a quote! It has a small fair pricing and gives you super easy access to the OPC UA technology.
